modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/utils/constants.es.js

```diff
@@ -30,6 +32,11 @@ export const NOT_OPERATORS = {
 	NOT_EQ: 'not-eq',
 };
 
+export const NOT_OPERATORS2 = {
```

Comment:
Maybe a more descriptive name?

Reply:
HAS_OPERATORS?

Reply:
Totally agree @cgoncas , I didn't have good ideas for the naming here 😅